### DISCUSSION TITLE
fat 32 does not allow more than 4gb

### DISCUSSION THREAD

**Discussion by daniel0torres** [Aurelio92/RVLoader#38] [General]
They should make it possible to use exfat, the reason is that I tried to install super smash bross brawl from wii but it weighs 7 gb, which means that it doesn't allow me to add it to the usb since fat 32 only allows files of less than 4gb. Do you know if partitioning the file I can run it? and thanks for the platform I have spent a lot of time :)

**Comment by Aurelio92** [comment]
I thought about integrating exFAT support, but, while simple, it would make people using this file system not able to run other homebrews, since they require the USB to be partitioned using FAT32.

If you only need to use wbfs files that are bigger than 4 GB, you can simply split the wbfs game into multiple files. You can do that with iso2wbfs or any of the many backup manager software out there
